deploy.go

```diff
@@ -136,7 +136,17 @@ func (p *deploycmd) deploy(c *cli.Context) error {
 func (p *deploycmd) deployFunc(c *cli.Context, funcFilePath string) error {
 	funcFileName := path.Base(funcFilePath)
 
-	err := c.App.Command("bump").Run(c)
+	ff, err := loadFuncfile()
+	if err != nil {
+		return err
+	}
+
+	err = validateImageName(ff.ImageName())
```

Comment:
in the case it's not a --local we would be getting this from dockerpush already, this just guards against bumping and building i guess, and seems like we're loading the function file twice now just to get the error before bump. anyway, nit...